apps/billing/payments/zibal_client.py

```
@@ -145,182 +155,150 @@ def request_payment(
-            
+
             return {
                 'track_id': validated.trackId,
                 'result': validated.result,
                 'message': validated.message,
                 'success': validated.result == self.RESULT_SUCCESS
             }
-            
+
         except ValidationError as e:
             logger.error("Invalid Zibal response format", exc_info=True)
```

Ruff (0.14.0)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)